include/triton/Conversion/TritonGPUToLLVM/Utility.h

```diff
@@ -778,10 +778,19 @@ emitBaseIndexForMfmaLayout(Location loc, RewriterBase &rewriter,
   Value laneId = urem(threadId, effectiveWarpSize);
 
   Value warpId = udiv(threadId, warpSize);
-  Value warpId0 =
-      urem(urem(warpId, warpsPerCTA[0]), i32_val(shape[0] / nonKDim));
-  Value warpId1 = urem(urem(udiv(warpId, warpsPerCTA[0]), warpsPerCTA[1]),
```

Comment:
So the default behavior before this change is the same as the transposed behavior ? I have a dot test that is failing once I started relying on `getIsTransposed` and `order`, the failing test has an mfma layout that is not transposed. But seems it was passing on `main` because the default behavior IS transposed. Any idea ?